wire-compiler/src/main/java/com/squareup/wire/java/TypeWriter.java

```diff
@@ -210,9 +210,9 @@ private TypeSpec toTypeSpec(WireMessage type) {
     for (WireField field : type.fieldsAndOneOfFields()) {
       TypeName fieldType = fieldType(field);
 
-      if (field.type().isScalar()
-          || javaGenerator.isEnum(field.type())
-          || field.isRepeated()) {
+      if ((field.type().isScalar() || javaGenerator.isEnum(field.type()))
```

Comment:
This change removes the emission of the default empty list constant which is no longer required. Scalars and enums still get defaults.